apps/wallet/src/ui/app/pages/home/transactions/index.tsx

```diff
@@ -1,19 +1,61 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
+
 import { memo } from 'react';
 
 import PageTitle from '_app/shared/page-title';
-import RecentTransactions from '_components/transactions-card/RecentTransactions';
-
-import st from './Transactions.module.scss';
+import { ErrorBoundary } from '_components/error-boundary';
+import Loading from '_components/loading';
+import { Transaction } from '_components/transactions-card/Transaction';
+import { useAppSelector, useGetTransactionsByAddress } from '_hooks';
+import Alert from '_src/ui/app/components/alert';
 
 function TransactionsPage() {
+    const activeAddress = useAppSelector(({ account: { address } }) => address);
+
+    const {
+        data: txns,
+        isError,
+        isLoading,
+    } = useGetTransactionsByAddress(activeAddress);
+
+    if (isError) {
+        return (
+            <div className="p-2">
+                <Alert mode="warning">
+                    <div className="mb-1 font-semibold">
+                        Something went wrong
```

Comment:
small nit - `mb-1` class here causes the text in the alert banner to be off-center

<img width="329" alt="image" src="https://user-images.githubusercontent.com/122397493/214238775-dedf81ba-f9e8-47f1-bd64-faf77e03c945.png">

Reply:
can also drop the container div and pass the `p-2` class into `<Alert />`

Reply:
done